InOut/CMakeLists.txt

```diff
@@ -157,25 +155,14 @@ if(USE_IPMIDI)
     make_plugin(ipmidi ipmidi.c "${ipmidi_LIBS}")
 endif()
 
-check_deps(USE_JACK JACK_HEADER JACK_LIBRARY)
+check_deps(USE_JACK JACK_HEADER JACK_LIBRARY JACK_INCLUDE_PATH)
 if(USE_JACK)
-    set(rtjack_LIBS "")
-    if(LINUX)
-        list(APPEND rtjack_LIBS
-            ${JACK_LIBRARY} ${PTHREAD_LIBRARY})
-    elseif(WIN32)
-        check_deps(JACK_INCLUDE_PATH)
-        include_directories("${JACK_INCLUDE_PATH}")
-        list(APPEND rtjack_LIBS
-            ${JACK_LIBRARY} ${PTHREAD_LIBRARY})
-    else()
-        list(APPEND rtjack_LIBS
-            ${JACK_LIBRARY} ${PTHREAD_LIBRARY})
-    endif()
```

Comment:
Why did you remove this code?

Reply:
Well, `list(APPEND rtjack_LIBS ${JACK_LIBRARY} ${PTHREAD_LIBRARY})` was in all 3 branches. And we'd need the jack include directory regardless because we `#include <jack/jack.h>`